src/main/java/com/ably/kafka/connect/client/DefaultAblyClient.java

```diff
@@ -59,6 +67,21 @@ public void publishFrom(SinkRecord record) throws ConnectException {
             //this exception should cause the calling task to abort
             throw new ConnectException("Cannot publish to Ably when connection failed");
         }
+        //check both channel and message config for skipping the record
```

Comment:
Horizontal spacing:

```suggestion
        // check both channel and message config for skipping the record
```

(though I do recognise there's an earlier comment in this same file where there isn't a space after the `//`, however I do think it's fairly conventional - for readability - to have a space here)